frontend/server/src/Controllers/User.php

```diff
@@ -2433,6 +2433,35 @@ public static function getFullRankByProblemsSolved(
         ];
     }
 
+    /**
+     * Gets the best users of the current month
+     *
+     * @return list<array{category: string, classname: string, coder_of_the_month_id: int, country_id: string, description: null|string, interview_url: null|string, problems_solved: int, ranking: int, school_id: int|null, score: float, selected_by: int|null, time: string, user_id: int, username: string}>
```

Comment:
```suggestion
     * @return list<CoderOfTheMonth>
```

y luego arriba puedes agregar

```php
/**
 * @psalm-type CoderOfTheMonth=array{category: string, classname: string, coder_of_the_month_id: int, country_id: string, description: null|string, interview_url: null|string, problems_solved: int, ranking: int, school_id: int|null, score: float, selected_by: int|null, time: string, user_id: int, username: string}
 */
```

y en todos los demás lugares donde se usa este tipo larguísimo puedes usar `CoderOfTheMonth`.

Reply:
Voy a subir el cambio con el error, porque no sé por qué está apareciendo.